pkg/metrics/metrics.go

```diff
@@ -48,16 +48,24 @@ var (
 			Help:      "Latency of an admission attempt, broken down by result.",
 		}, []string{"result"},
 	)
+
+	PendingWorkloads = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Subsystem: subsystemName,
+			Name:      "pending_workloads",
+			Help:      "Number of pending workloads, per queue and cluster_queue.",
+		}, []string{"cluster_queue", "queue"})
 )
 
 func AdmissionAttempt(result AdmissionResult, duration time.Duration) {
 	admissionAttempts.WithLabelValues(string(result)).Inc()
 	admissionAttemptLatency.WithLabelValues(string(result)).Observe(duration.Seconds())
 }
 
-func init() {
+func Register() {
 	metrics.Registry.MustRegister(
 		admissionAttempts,
 		admissionAttemptLatency,
+		PendingWorkloads,
```

Comment:
What about lowcase `PendingWorkloads` to `pendingWorkloads`  and move  `reportPendingWorkloads` to this package and pass in the necessary parameters.